lighthouse-core/gather/computed/metrics/lantern-first-cpu-idle.js

```diff
@@ -56,6 +56,7 @@ class LanternFirstCPUIdle extends LanternInteractive {
       longTasks.push({start: timing.startTime, end: timing.endTime});
     }
 
+    longTasks.sort((a, b) => a.start - b.start);
```

Comment:
same question. can we do the sort earlier in a more generic location?

I want to avoid the situation where someone writes a new audit and forgets they gotta sort on their own.